cmd/agent/app/options/options.go

```diff
@@ -192,6 +194,7 @@ func NewGrpcProxyAgentOptions() *GrpcProxyAgentOptions {
 		CaCert:                    "",
 		ProxyServerHost:           "127.0.0.1",
 		ProxyServerPort:           8091,
+		HealthServerAddr:          "",
```

Comment:
naming nit: HealthServerHost (to be consistent with ProxyServerHost)

Reply:
The ProxyServerHost is a host to connect to, as opposed to an address to listen on. One doesn't really bind a listener to a `Host` IMHO

Reply:
Yes, but it seems to me that an address is composed of a host + port, and since this flag only covers the former, it is better named "~host".

Reply:
Updated